src/test/resources/com/puppycrawl/tools/checkstyle/checks/coding/unnecessaryparentheses/InputUnnecessaryParenthesesIfStatement.java

```diff
@@ -130,7 +130,7 @@ public void checkBooleanStatements() {
         if (( // violation
                 a!=(b==c) && (a // violation
                         && (b==c))) // violation
-                 || (a || a!=(b<=c))     // ok
+                 || (a || a!=(b<=c))     // violation
```

Comment:
Please add to comment which parenthesis are unnecessary since there are two sets of unnecessary parenthesis on this line, maybe something like `violation, outer parenthesis are unnecessary`.

Reply:
Unless the test specifically requires specific lines, I found in previous PRs that it makes where the violation is clearer if the `()`s are on different lines.

Reply:
Yes, that is better, changed it to that format.

Reply:
Resolved